wait-for-deployment

```diff
@@ -11,6 +11,7 @@
 set -o errexit
 set -o pipefail
 set -o nounset
+set -m
 
```

Comment:
this enables job control which is otherwise only enabled in interactive mode http://unix.stackexchange.com/a/196606/73578